x/feegrant/client/cli/tx.go

```diff
@@ -125,25 +125,24 @@ Examples:
 					return err
 				}
 
-				if periodClock > 0 && periodLimit != nil {
-					periodReset := getPeriodReset(periodClock)
-					if exp != "" && periodReset.Sub(expiresAtTime) > 0 {
-						return fmt.Errorf("period(%d) cannot reset after expiration(%v)", periodClock, exp)
-					}
-
-					periodic := feegrant.PeriodicAllowance{
-						Basic:            basic,
-						Period:           getPeriod(periodClock),
-						PeriodReset:      getPeriodReset(periodClock),
-						PeriodSpendLimit: periodLimit,
-						PeriodCanSpend:   periodLimit,
-					}
-
-					grant = &periodic
-
-				} else {
-					return fmt.Errorf("invalid number of args %d", len(args))
+				if periodLimit == nil {
+					return fmt.Errorf("Period limit was not set")
 				}
+
+				periodReset := getPeriodReset(periodClock)
```

Comment:
The following code is no longer indented under the if check.  I think it looks cleaner this way, but am happy to change it back to being under an if check with an else for the error case if that is preferred.